deeppavlov/models/torch_bert/torch_bert_sequence_tagger.py

```diff
@@ -0,0 +1,375 @@
+            if idx.numel() > 0:
+                d = data_.view(idx.numel(), -1)
+                k = 0
+                for idx_ in idx:
+                    res[idx_] = d[k].to(torch.float64)
+                    k += 1
+        return res
+
+    tensor_flat = torch.stack(dynamic_stitch([word_indices_flat, nonword_indices_flat], [elements, paddings]))
+    # tensor_flat -> [x, x, x, x, x, 0, x, 0, 0]
+
+    tensor = torch.reshape(tensor_flat, (batch_size, max_token_seq_len.item(), nf_int))
+    # tensor -> [[x, x, x],
+    #            [x, x, 0],
+    #            [x, 0, 0]]
+
+    return tensor
+
+
+def token_labels_to_subtoken_labels(labels, y_mask, input_mask):
+    subtoken_labels = []
+    labels_ind = 0
+    n_tokens_with_special = int(np.sum(input_mask))
+
+    for el in y_mask[1:n_tokens_with_special - 1]:
+        if el == 1:
+            subtoken_labels += [labels[labels_ind]]
+            labels_ind += 1
+        else:
+            subtoken_labels += [labels[labels_ind - 1]]
+
+    subtoken_labels = [0] + subtoken_labels + [0] * (len(input_mask) - n_tokens_with_special + 1)
+    return subtoken_labels
+
+
+@register('torch_bert_sequence_tagger')
+class TorchBertSequenceTagger(TorchModel):
+    """BERT-based model on PyTorch for text tagging. It predicts a label for every token (not subtoken) in the text.
+    You can use it for sequence labeling tasks, such as morphological tagging or named entity recognition.
+
+    Args:
+        n_tags: number of distinct tags
+        pretrained_bert: pretrained Bert checkpoint path or key title (e.g. "bert-base-uncased")
+        return_probas: set this to `True` if you need the probabilities instead of raw answers
+        bert_config_file: path to Bert configuration file, or None, if `pretrained_bert` is a string name
+        attention_probs_keep_prob: keep_prob for Bert self-attention layers
+        hidden_keep_prob: keep_prob for Bert hidden layers
+        optimizer: optimizer name from `torch.optim`
+        optimizer_parameters: dictionary with optimizer's parameters,
+                              e.g. {'lr': 0.1, 'weight_decay': 0.001, 'momentum': 0.9}
+        learning_rate_drop_patience: how many validations with no improvements to wait
+        learning_rate_drop_div: the divider of the learning rate after `learning_rate_drop_patience` unsuccessful
+            validations
+        load_before_drop: whether to load best model before dropping learning rate or not
+        clip_norm: clip gradients by norm
+        min_learning_rate: min value of learning rate if learning rate decay is used
+    """
+
+    def __init__(self,
+                 n_tags: int,
+                 pretrained_bert: str,
+                 bert_config_file: Optional[str] = None,
+                 return_probas: bool = False,
+                 attention_probs_keep_prob: Optional[float] = None,
+                 hidden_keep_prob: Optional[float] = None,
+                 optimizer: str = None,
```

Comment:
Defalt value for base class is `AdamW`. Won't default value `None` cause Error in base class?

Reply:
will.fixed